docs/examples.md

```diff
@@ -0,0 +1,140 @@
+---
+sidebar_label: Examples
+title: 'Code examples'
+---
+
+## Passing an input to the Actor
+
+The fastest way to get results from an Actor is to pass input directly to the `call` function.
+We can set up the input, pass it to `call` function and get the reference of running Actor (or wait for finish).
+
+```python
+from apify_client import ApifyClient
+
+# Client initialization with the API token
+apify_client = ApifyClient(token='MY_APIFY_TOKEN')
+
+actor_client = client.actor('apify/instagram-hashtag-scraper')
+
+input_data = { 'hashtags': ['rainbow'], 'resultsLimit': 20 }
+
+# Run the Actor and wait for it to finish up to 60 seconds.
+# Input is not persisted for next runs.
+run_data = actor_client.call(run_input=input_data, timeout_secs=60)
```

Comment:
We need to try to run the examples all the time, same thing happened to us with the getting started examples in JS client - this will never work because there is no `client` defined in this code

e.g.: this won't work

`actor_client = client.actor('apify/instagram-hashtag-scraper')`